examples/flask/example.py

```diff
@@ -18,3 +18,28 @@ def index():
 @picobox.pass_('magic')
 def magic(magic):
     return jsonify({'magic': magic})
+
+
+@app.before_request
+def serve_eggs_with_spam():
+    box = picobox.Box()
+
+    # on requests to /eggs, override the value of magic with 'spam'
+    if request.path == '/eggs':
+        box.put('magic', 'spam')
+
+    picobox.push(box, chain=True)
+
+
+@app.after_request
+def take_spam_away(response):
+    # pop the box from the top of the stack to remove the override
+    picobox.pop()
+
```

Comment:
nit: Could you please remove this empty line? :)

Reply:
Done!